packages/svelte/tests/runtime-runes/samples/await-pending-destroy/_config.js

```diff
@@ -0,0 +1,31 @@
+import { flushSync } from 'svelte';
+import { test } from '../../test';
+
+export default test({
+	async test({ component, assert, target, logs }) {
+		// Set a promise on the component
+		const p1 = new Promise((resolve) => setTimeout(resolve, 32));
```

Comment:
Why this magic numbers here? Less than that was not enough to make it happen?

Reply:
It's completely random ! 

It's true that it's not great.
I just rewrite that using a polyfill of Promise.withResolver()